app/models/location.rb

```diff
@@ -1,4 +1,6 @@
 class Location < ActiveRecord::Base
-  attr_accessible :category, :comments, :description, :lat, :lon, :name, :status
+  attr_accessible :category, :comments, :description, :department_id,
+                  :floor, :container, :lat, :lon, :name, :status
```

Comment:
the non-department stuff probably shouldn't be in this pr. will merge anyway but...